apps/deploy-web/env/.env

```
@@ -25,7 +25,7 @@ BASE_API_SANDBOX_URL=$NEXT_PUBLIC_BASE_API_SANDBOX_URL
 
 NEXT_PUBLIC_NODE_ENV=$NODE_ENV
 
-NEXT_PUBLIC_PROVIDER_PROXY_URL=/provider-proxy
+NEXT_PUBLIC_PROVIDER_PROXY_URL="/provider-proxy-%{NETWORK}"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)